account_bank_statement_import_move_line/views/account_bank_statement_view.xml

```diff
@@ -11,7 +11,8 @@
                 <button name="%(account_statement_line_create_action)d"
                         string="Import Journal Items" type="action"
                         class="oe_highlight"
-                        attrs="{'invisible':[('state', '!=', 'open')]}"/>
+                        attrs="{'invisible':[('state', '!=', 'open')]}"
+                        groups="account.group_account_user"/>
```

Comment:
Why adding this group?

Reply:
That's a good question, tested with admin you could see Invoicing menus and this button but my surprise when I see fails on create new journal items and system pop up this group name.
So my though was why I should see an action that could frustrate user with some message "sorry but you can't"

Reply:
OK, let's keep it this way then.